src/form-builder/components/FormBuilder.jsx

```diff
@@ -68,6 +76,66 @@ export default class FormBuilder extends Component {
     }, commonConstants.toastTimeout);
   }
 
+  getValidJsonFileNames(fileNames) {
+    const validJsonFiles = [];
+    fileNames.forEach(fileName => {
+      if (fileName.endsWith('.json')) {
+        validJsonFiles.push(fileName);
+      } else {
+        this.updateImportErrors(fileName, 'Invalid file format');
+      }
+    });
+    return validJsonFiles;
+  }
+
+  validateAndLoadZipFile(jsonZip) {
+    const self = this;
+    const maxAllowedSize = 500 * 1024;
```

Comment:
can we make this configurable? or larger? I see today most of our forms are larger that 500KB. So we are unable to import a single form also.

Reply:
Increased to 5MB

Reply:
Done. Increased to 5MB